packages/webapp/src/pages/members/index.tsx

```diff
@@ -9,24 +9,22 @@ import {
     SideNavLayout,
     Heading,
     PaginationNav,
-    queryMembersStats,
-    queryMembers,
     queryNewMembers,
+    usePagedMembers,
+    LoadingContainer,
+    Text,
 } from "_app";
-import { MemberChip, MembersGrid } from "members";
+import { MemberChip, MemberData, MembersGrid } from "members";
 
 const MEMBERS_PAGE_SIZE = 18;
 const NEW_MEMBERS_PAGE_SIZE = 12;
 
 export const getServerSideProps: GetServerSideProps = async ({ query }) => {
     const queryClient = new QueryClient();
 
-    const membersPage = parseInt((query.membersPage as string) || "1");
     const newMembersPage = parseInt((query.newMembersPage as string) || "1");
 
     await Promise.all([
-        queryClient.prefetchQuery(queryMembersStats),
-        queryClient.prefetchQuery(queryMembers(membersPage, MEMBERS_PAGE_SIZE)),
```

Comment:
question: we actually never leveraged our server side rendering mechanism to generate seo tags, page titles, nice google pages previews etc. should we refactor all of our pages to not rely on server side rendering anymore? do we want the community to be discoverable?

Reply:
I think we _do_ want the community to be discoverable, but I think that we will de facto remove the SSR stuff as we move to box subchain data and away from RPC calls. We can still leverage it in the future for SEO though. (as we discussed)